packages/did-provider-oyd/README.md

```diff
@@ -0,0 +1,4 @@
+# OwnYourData did:oyd provider
+
+This package contains an implementation of `AbstractIdentifierProvider` for the `did:oyd` method.
+This enables creation and control of `did:oyd` entities.
```

Comment:
can you add a link to the spec for `did:oyd`?

Reply:
referenced spec